scss/components/_soe_images.scss

```diff
@@ -54,9 +54,14 @@
       @include breakpoint-max(medium) {
         height: auto;
       }
-
       .logged-in & {
         height: calc(100vh - 235px);
+        @include breakpoint-max(small) {
```

Comment:
@cjwest It sounds from both you and Andy like this change is not coming from a client request.  It seems small, but still takes quite a bit of time away from our work already estimated for this sprint, not only in creating and documenting the change, but also testing, releasing, and then re-testing that it does not affect anything else.  If you can revert this file and commit the resulting css, I'll go ahead and merge the changes related to the margin-bottom on hero banners.